editor-components/feature/feature3/feature3.tsx

```diff
@@ -1,105 +1,205 @@
 import * as React from "react";
 import { BaseFeature } from "../../EditorComponent";
 import styles from "./feature3.module.scss";
+import { ComposerIcon } from "../../../composer-base-components/icon/icon";
 
-type ISimpleBoxes = {
-  title: string;
-  description: string;
+type Logos = {
   image: string;
+  url:string;
 };
+
+type Cards = {
+  title: string;
+  icon: string;
+}
+
+
 class Feature3 extends BaseFeature {
   constructor(props?: any) {
     super(props, styles);
 
-    
     this.addProp({
       type: "array",
-      key: "simple-boxes-content",
-      displayer: "Simple Boxes Content",
+      key: "logos",
+      displayer: "Logo Images",
       value: [
         {
-          type: "object",
-          key: "simple-boxes",
-          displayer: "Simple Boxes",
+          type:"object",
+          key:"logo",
+          displayer: "Logo",
           value: [
             {
-              type: "string",
-              key: "title",
-              displayer: "Title",
-              value: "Bill Gates",
+              type: "page",
+              key: "link",
+              displayer: "Logo Link",
+              value: "",
             },
             {
-              type: "string",
-              key: "description",
-              displayer: "Description",
-              value:"Bill Gates is a prominent American entrepreneur and philanthropist who co-founded Microsoft Corporation, one of the world's largest software companies. He is also known for his significant contributions to the development of computers."
+              type: "image",
+              key: "image",
+              displayer: "Logo Image",
+              value: "https://template-kit.evonicmedia.com/layout49/wp-content/uploads/2023/10/new-logoipsum-289.png",
```

Comment:
Images need to be uploaded to the Spica storage, and the values must be retrieved from the storage URL.